elements/designer-mini-nav/designer-mini-nav.html

```diff
@@ -47,6 +47,46 @@
 
   Polymer({
     is: 'designer-mini-nav',
+
+    properties: {
+
+      expanded: {
+        type: Boolean,
+        value: false,
+        reflectToAttribute: true,
+      },
+
+      _collapsing: {
+        type: Boolean,
+        value: function() { return false; },
+      },
+
+    },
+
+    listeners: {
+      'designer-mini-nav-collapse': '_onCollapse',
```

Comment:
I'm going to YAGNI this for now. I don't feel like figuring out the general pattern for "some of my decedents have an operation which should trigger me to collapse" that doesn't involve said decendent explicitly cooperating.